apps/expo/src/lang/nl-NL.json

```diff
@@ -152,15 +152,22 @@
           }
         },
         "bluetooth": {
-          "enabling": "Bluetooth inschakelen...",
+          "enabling": "Bluetooth is momenteel uitgeschakeld. Om verder te kunnen gaan, dient u uw Bluetooth in te schakelen",
           "alert": {
             "title": "Bluetooth toestemming",
-            "message": "Deze app gebruikt Bluetooth, maar registreert uw locatie niet. Op Android valt Bluetooth onder locatiediensten. Daarom krijgt u zo een vraag over locatie.",
-            "button": "Oké, vraag maar!"
+            "message": "Deze app maakt gebruik van Bluetooth, maar registreert uw locatie niet. Op Android valt Bluetooth onder locatiediensten. Daarom zult u binnenkort een vraag krijgen over locatietoegang. Bovendien dient u toestemming te geven aan apparaten in de buurt en locatie.",
```

Comment:
This line shows on iOS too.
iOS only needs Bluetooth and Location and we probably should not mention Android.